ui/packages/consul-ui/mock-api/v1/internal/ui/service-topology/_

```diff
@@ -98,6 +99,7 @@ ${
         const allowed = hasPerms ? false : fake.random.boolean();
         return `
     {
+      "Kind": "${fake.helpers.randomize(['mesh-gateway', 'ingress-gateway', 'terminating-gateway', '', undefined])}",
```

Comment:
Could we use this form instead here, it makes it more or a truer mock of the actual API, I'm pretty sure the API never returns actual `undefined` as a value.

```javascript
${fake.booleanOfSomeSort() ? `
  "Kind": "${fake.helpers.randomize(['mesh-gateway', 'ingress-gateway', 'terminating-gateway', ''])}",
` : ``}
```

(fake.booleanOfSomeSort() doesn't exist, but maybe look up what fits best here faker has a few ways to do this kind of thing, i.e. do you want it to be 50/50 of the time or 10/90 etc)

Reply:
Update to show `Kind` 70% of the time.